app/models/stripe_card.rb

```diff
@@ -354,8 +354,8 @@ def all_hcb_codes
     canonical_transaction_hcb_codes + canonical_pending_transaction_hcb_codes
   end
 
-  def hcb_codes
-    @hcb_codes ||= ::HcbCode.where(hcb_code: all_hcb_codes).includes(:tags)
+  def local_hcb_codes
+    HcbCode.where(hcb_code: all_hcb_codes).includes(:tags)
   end
```

Comment:
We're not entirely sure whether the caching and `::` can be removed at this time
```suggestion
  def local_hcb_codes
    @local_hcb_codes ||= ::HcbCode.where(hcb_code: all_hcb_codes).includes(:tags)
  end
```